package.json

```diff
@@ -1,19 +1,34 @@
 {
-  "name": "camera-widget-for-phonegap",
+  "name": "CameraWidgetForPhoneGap",
   "version": "3.2.0",
   "description": "Enable native camera functionality within your Mendix hybrid app",
-  "repository": {
```

Comment:
Why did you remove `repository`

Reply:
It wasnt originally there but i've added it.